www/src/pages/foundations/typography.tsx

```diff
@@ -42,356 +101,233 @@ const measuredTypeProps = {
-            </tr>
-            <tr>
-              <td colSpan={2}>
-                <MeasuredItem {...measuredTypeProps}>
-                  <p className="lead m-0">Large Body</p>
-                </MeasuredItem>
-              </td>
-              <td>
-                <code>.lead</code>
-              </td>
-            </tr>
-            <tr>
-              <td colSpan={3}>
-                <MeasuredItem {...measuredTypeProps}>
-                  <p className="m-0">Body</p>
-                </MeasuredItem>
-              </td>
-            </tr>
-            <tr>
-              <td colSpan={2}>
-                <MeasuredItem {...measuredTypeProps}>
-                  <p className="small m-0">Small Body</p>
-                </MeasuredItem>
-              </td>
-              <td>
-                <code>.small</code>
-              </td>
-            </tr>
-            <tr>
-              <td colSpan={2}>
-                <MeasuredItem {...measuredTypeProps}>
-                  <p className="x-small m-0">Extra Small Body</p>
-                </MeasuredItem>
-              </td>
-              <td>
-                <code>.x-small</code>
-              </td>
-            </tr>
-            <tr>
-              <td colSpan={2}>
-                <MeasuredItem {...measuredTypeProps}>
-                  <p className="micro m-0">Micro Body</p>
-                </MeasuredItem>
-              </td>
-              <td>
-                <code>.micro</code>
-              </td>
-            </tr>
-          </tbody>
-          <tbody>
-            <tr>
-              <th colSpan={3}>
-                <h2 className="mt-3">Display</h2>
-              </th>
-            </tr>
-            <tr>
-              <th>Desktop</th>
-              <th>Mobile</th>
-              <th>CSS Class</th>
-            </tr>
-            {[1, 2, 3, 4].map(displaySize => (
-              <tr key={displaySize}>
-                <td>
+                ),
+              },
+              {
+                Header: 'Mobile',
+                Cell: ({ row }: DataTableTypes) => (
+                  <div className="mobile-type">
+                    <MeasuredItem {...measuredTypeProps}>
+                      <p className={`m-0 h${row.original.size}`}>
+                        Heading {row.original.size}
+                      </p>
+                    </MeasuredItem>
+                  </div>
+                ),
+              },
+              {
+                Header: 'CSS Class',
+                Cell: ({ row }: DataTableTypes) => (
+                  <code style={{ fontSize: '16px' }}>
+                    .h{row.original.size}
+                  </code>
+                ),
+              },
+            ]}
+          >
+            <DataTable.Table />
+          </DataTable>
+        </div>
+
+        <h2 className="mb-2">Body</h2>
+        <div className="mb-4">
+          <DataTable
+            itemCount={2}
+            data={bodyClassesAndDescriptions.map(({
```

Comment:
no need to do `.map` here, `data={bodyClassesAndDescriptions}` should work

Reply:
Thanks, done